src/components/EnterBarcodePage/CompleteBarcodeButton.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import { makeStyles } from "@material-ui/core/styles";
+
+const CompleteBarcodeButton = ({
+  handleClick,
+}: {
+  handleClick: () => any | null;
+}) => {
+  const classes = useCompleteBarcodeButtonStyles();
+  return (
+    <Button
+      variant="contained"
+      color="secondary"
+      classes={{
+        root: classes.root,
+      }}
+      type="submit"
+      onClick={handleClick}
+    >
+      Next
+    </Button>
+  );
+};
+
+const useCompleteBarcodeButtonStyles = makeStyles({
```

Comment:
once linted, this will need to be moved up so it's defined before we're used - in general this is the pattern we'll start to enforce from now on